libs/liblobby/lobby/interface.lua

```diff
@@ -191,6 +191,54 @@ local function UpdateAndCreateMerge(userData, status)
 	return battleStatus, updated
 end
 
+--n = pow(2,i) -- where i = 0,31
+--print('{',n//1000000,',', n%1000000,'},')
+local bin2decmillion16 = { -- the <1M and >1M parts of 2^nth powers where n > 16
+	{ 0 , 65536 }, --16
+	{ 0 , 131072 }, --17
+	{ 0 , 262144 }, --18
+	{ 0 , 524288 }, --19
+	{ 1 , 48576 },
+	{ 2 , 97152 },
+	{ 4 , 194304 },
+	{ 8 , 388608 },
+	{ 16 , 777216 }, --24
+	{ 33 , 554432 }, --25
+	{ 67 , 108864 }, --26
+	{ 134 , 217728 }, --27
+	{ 268 , 435456 }, --28
+	{ 536 , 870912 },  --29
+	{ 1073 , 741824 }, --30
+	{ 2147 , 483648 }, -- 31
+	}
+
+-- Combine two 16 bit numbers into a string-formatted 32-bit integer
+local function lsbmsb16tostring(lsb,msb)
```

Comment:
I'd like this code to be unit tested. I don't trust human reviewers/authors to be correct here (as history has shown).

I'm not sure if current Chobby has unit testing framework setup, but luaunit is popular and not that difficult to add. I've used it myself to test a promise library I wrote, see https://github.com/gajop/spring-lua-utils/blob/c1ed690f660c8f46edd3a35f6bceca19eab2c79b/promise.lua

On code style, I think this doesn't match Chobby's syntax. I believe Chobby uses camelCase for variables and this is all lowercase.